src/lib/CookieJar.js

```diff
@@ -60,12 +57,12 @@ class CookieJar {
         if (!keys) return []; // Make TS happy (match() can potentially return NULL)
 
         return keys.map(key => {
-            const type = parseInt(key[0]);
+            const type = parseInt(key[0], 10);
             const userFriendlyAddress = Nimiq.Address.fromBase64(key.substr(1)).toUserFriendlyAddress();
 
             return /** @type {KeyInfo} */ ({
                 userFriendlyAddress,
-                type
+                type,
             });
```

Comment:
I'm surprised that the linter does not inline this one - seeing that it inlined map/filter lines with lambdas. I'd prefer it inlined

Reply:
The linter wants a line-break after each opening curly bracket `{`.

Reply:
The object construction was not _changed_ by ESLint, but was written that way by a human. ESLint just added the trailing comma.